packages/react-canvas-core/src/entities/layer/TransformLayerWidget.tsx

```diff
@@ -59,8 +60,10 @@ export class TransformLayerWidget extends React.Component<TransformLayerWidgetPr
 	}
 
 	render() {
+		const svgLayerStyles = this.props.svgLayerStyles || {};
+
 		if (this.props.layer.getOptions().isSvg) {
-			return <S.SvgLayer style={this.getTransformStyle()}>{this.props.children}</S.SvgLayer>;
+			return <S.SvgLayer style={{ ...svgLayerStyles, ...this.getTransformStyle()}}>{this.props.children}</S.SvgLayer>;
```

Comment:
would actually prefer this to be done via `className={this.props.className}`

Reply:
the point is to have possibility dynamically add background and sizes depends on image size

Reply:
Yeah so, ideally what we actually want to do here is let the developer render a new simple layer behind the nodes (and) links (which can be achieved using the api's in react-canvas)

Reply:
(which can be dynamic)

Reply:
Is there a sample? Also, I set an image as a background to keep all zooming, dragging functionality works